diff --git a/node_modules/@microsoft/signalr/dist/esm/HubConnection.js b/node_modules/@microsoft/signalr/dist/esm/HubConnection.js
index eec6006..254659b 100644
--- a/node_modules/@microsoft/signalr/dist/esm/HubConnection.js
+++ b/node_modules/@microsoft/signalr/dist/esm/HubConnection.js
@@ -502,7 +502,7 @@ export class HubConnection {
                         try {
                             await this._sendMessage(this._cachedPingMessage);
                         }
-                        catch {
+                        catch(e) {
                             // We don't care about the error. It should be seen elsewhere in the client.
                             // The connection is probably in a bad or closed state now, cleanup the timer so it stops triggering
                             this._cleanupPingTimer();
